drivers/ovsdriver.go

```diff
@@ -88,10 +88,13 @@ func (d *OvsDriver) getIntfName() (string, error) {
 		// Pick next port number
 		d.oper.CurrPortNum++
 		intfName := fmt.Sprintf("vport%d", d.oper.CurrPortNum)
+		intfName2 := fmt.Sprintf("vvport%d", d.oper.CurrPortNum)
```

Comment:
Since the name of veth pair is passed between different layers of the driver, we pass only one and derive the other one as needed..